CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Added
 
-- Implemented `Eq` for `common::headers::Encoding`, `common::headers::MediaType`, 
+- Implemented `Eq` for `common::headers::Encoding`, `common::headers::MediaType`,
   `common::headers::Headers`, `common::HttpHeaderError`, `common::Body`, `common::Version`,
   `common::RequestError`, `request::Uri`, `request::RequestLine`, `response::StatusCode`,
   `response::ResponseHeaders`
```

Comment:
I don't see this in main. Is this part of another commit? 

More generally? Is this relevant for the release notes?

Reply:
I can see it on main. My editor just removed a trailing space.
https://github.com/firecracker-microvm/micro-http/blame/main/CHANGELOG.md#L5

Reply:
omitted the removal of a trailing space from this PR.